data-science-onramp/data-ingestion/setup.sh

```diff
@@ -0,0 +1,9 @@
+# Submit a PySpark job via the Cloud Dataproc Jobs API
+# Requires having CLUSTER_NAME and BUCKET_NAME set as 
+# environment variables
+
+gcloud dataproc jobs submit pyspark \
+    --cluster ${CLUSTER_NAME} \
+    --jars gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar \
```

Comment:
@bradmiro how often would this jar change? I'm worried about this shell script going stale when the jar is updated

Reply:
Not often at all. I haven't found a reliable way to dynamically call this jar, but support for new Scala versions is not frequent.